index.js

```diff
@@ -36,14 +37,18 @@ const createRetrieval = async (req, res, client) => {
     INSERT INTO retrievals (
       retrieval_template_id,
       spark_version,
-      zinnia_version
+      zinnia_version,
+      created_at_round,
+      created_from_address
     )
-    VALUES ($1, $2, $3)
+    VALUES ($1, $2, $3, $4, $5)
     RETURNING id
   `, [
     retrievalTemplate.id,
     meta.sparkVersion,
-    meta.zinniaVersion
+    meta.zinniaVersion,
+    round,
+    req.connection.remoteAddress
```

Comment:
What are the implication for user's privacy when we start collecting their IP addresses?

Reply:
looks like we need to add a privacy policy https://termly.io/faq/is-ip-address-personal-information-under-ccpa/#:~:text=IP%20addresses%20are%20classified%20as,this%20in%20a%20privacy%20policy., cc @coreymjames